EventsExpress/ClientApp/src/containers/edit-event.js

```diff
@@ -62,13 +69,13 @@ const mapStateToProps = (state) => ({
 
 const mapDispatchToProps = (dispatch) => {
     return {
-        edit_event: (data) => dispatch(edit_event(data)),
+        edit_event: (data, onError, onSuccess) => dispatch(edit_event(data), onError, onSuccess),
```

Comment:
Since onError is not tied to an instance of a class - you can define it directly in this mapdispatchtoprops and remove the onSuccess parameter.